tests/smoke_tests/test_cluster_job.py

```diff
@@ -493,8 +493,8 @@ def test_inferentia():
     test = smoke_tests_utils.Test(
         'test_inferentia',
         [
-            f'sky launch -y -c {name} -t inf2.xlarge -- echo hi',
-            f'sky exec {name} --gpus Inferentia:1 echo hi',
+            f'sky launch -y -c {name} -t inf2.xlarge --cloud aws -- echo hi',
```

Comment:
let's avoid cloud here to test inferring cloud from instance type
```suggestion
            f'sky launch -y -c {name} -t inf2.xlarge -- echo hi',
```